modules/dynamic-plug-in-development.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins-get-started.adoc
+
+:_content-type: PROCEDURE
+[id="dynamic-plugin-development_{context}"]
+= Dynamic plug-in development
+
+You can run the plug-in using a local development environment. The {product-title} web console runs in a container connected to the cluster you have logged into.
+
+.Prerequisites
+* You must have an OpenShift cluster running.
+* You must have the OpenShift CLI (`oc`) installed.
+* You must have link:https://yarnpkg.com/[`yarn`] installed.
+* You must have link:https://www.docker.com/[Docker] v3.2.0 or newer or link:https://podman.io/[Podman] installed.
+
+.Procedure
+
+. In your terminal, run the following command to install yarn.
+
++
+[source,terminal]
+----
+yarn install
+----
+
+. After installing, run the following command to start yarn.
+
++
+[source,terminal]
+----
+yarn run start
+----
+
+. In another terminal window, login to the {product-title} CLI. 
```

Comment:
shall it be `login to the {product-title} `via` CLI` ?

Reply:
It should be 'through' we can't use via. Thanks!